src/ci/resources/enable_metadata_deleting.inc.conf

```
@@ -0,0 +1,14 @@
+# To use this partial config to activate deleting, include
+# it at the top of another config file (eg papi_v2_shared_application.inc.conf)
+
+services.MetadataService {
+  config {
+    delete-metadata {
+      # How long to pause between deletion attempts which fail, or if there's nothing to delete:
+      backoff-interval = 10 seconds
+
+      # How long to require after workflow completion before going ahead with deletion:
+      deletion-delay = 10 minutes
```

I have inadvertently belabored this out of proportion but my previous inline suggestion to remove the scary message included an accompanying factual reduction in scariness
```suggestion
      deletion-delay = 365 days
```

---

This one I will push back on a little, since this is a test config file. If you want me to add an _example_ config file with a less scary value set, that's fine, but this one should stay in the "here's a value to test this functionality out" column IMO